source/_integrations/bsblan.markdown

```diff
@@ -38,7 +38,11 @@ After completing the configuration flow, the BSBLan Climate integration will be
 available.
 
 For more documentation of the BSBLan device, check the [manual](https://1coderookie.github.io/BSB-LPB-LAN_EN/).
-The device support quite a few controllers from different manufacturers.
-[list of supported models](https://1coderookie.github.io/BSB-LPB-LAN_EN/chap03.html#313-other-manufacturers)
 
-The integration is tested with firmware latest stable version `v0.43`.
+**_To see a more detailed listing of the reported systems which are successfully used with BSB-LAN please follow the corresponding link:_**
+
+- **[`Brötje`](https://1coderookie.github.io/BSB-LPB-LAN_EN/chap03.html#311-broetje)**
```

Comment:
I think it's not necessary to make this bold

Reply:
indeed

Reply:
@liudger That include the list of authors 😉

Reply:
done 😊